README.md

```diff
@@ -104,10 +104,10 @@ mvn clean verify
 
 If you are a maintainer, you can release a new version by doing the following:
 
-- Merge the changes the need to be released into the `master` branch
+- Merge the changes they need to be released into the `master` branch
 - Checkout on to master locally and pull the latest changes
 - Run `mvn release:prepare`, this will generate 2 commits that will bump the version of the github-java-client
-- Push these changes to master
+- Push these changes to the master
```

Comment:
We either talk about pushing changes `to master` or `to the master branch`. You can consider changing the whole list to either one or the other option to keep consistency.

Reply:
This is still very confusing... Can you please address this? Probably just revert your change.